server/db/models/webpageCapture.js

```diff
@@ -38,7 +37,7 @@ var schema = new mongoose.Schema({
 //     return encryptPassword(candidatePassword, this.salt) === this.password;
 // });
 
-mongoose.model('Image', schema);
+mongoose.model('WebpageCapture', schema);
```

Comment:
Also need a diffingImg model to store previous and current screenshots for creating diffing image